code/_onclick/item_attack.dm

```diff
@@ -111,7 +111,7 @@
 	var/force_modifier = 0
 	if(force >= 5)
 		if(HAS_TRAIT(user, TRAIT_PANICKED_ATTACKER) || HAS_TRAIT(user, TRAIT_GHOULMELEE))
-			force_modifier = (-force * 0.8) // You do 20% damage cus ur scared
+			force_modifier = (-force * 0.2) // You do 80% damage cause you're a walking corpse
```

Comment:
This would also make adrenaline do 20% less damage instead of 80% less damage. You'll want to separate this into two different trait checks.

Reply:
Oh my god I didn't even realise they paired it with some other trait, why even do that. I'll fix it tomorrow I need to sleep, thanks for pointing it out.